CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 ### Other changes
 
  * Support for detecting features which have changed slightly during a re-import from a data source without a primary key, and reimporting them with the same primary key as last time so they show as edits as opposed to inserts. [#212](https://github.com/koordinates/sno/issues/212)
+ * `reset` now behaves more like `git reset` - specifically, `sno reset COMMIT` stays on the same branch but sets the branch tip to be `COMMIT`. [#60](https://github.com/koordinates/sno/issues/60)
```

Comment:
I'd put this at the top in a `Breaking changes` section. Maybe along with the datasets-v1 stuff

Reply:
Done